simplicity-sys/src/c_jets/jets_ffi.rs

```diff
@@ -1,6 +1,6 @@
 /* This file has been automatically generated. */
 
-use std::ffi::c_void;
+use crate::ffi::c_void;
```

Comment:
In 8e7f59bcbdac781c0472d1643ef246436ab59e15:

You are not allowed to change this file directly. It needs to be changed upstream and then pulled in by the vendoring script.